docs/en/deploy/Install-Alluxio-On-Docker.md

```diff
@@ -0,0 +1,618 @@
+  1. Only the specified ports (`-p` option) are exposed to the outside network, where the client may be run.
+     The command `-p <host-port>:<container-port>` maps the container port to a host port.
+     Therefore, you must explicitly expose the two ports 19999 and 19998 for the master container and the port
+     29999 and 30000 for the worker container.
+     Otherwise, the clients can't communicate with the master and worker.
+  1. You can refer to the master either by the container name
+     (`alluxio-master` for master container and `alluxio-worker` for worker container)  or
+     by the Docker host's IP address `$(hostname -i)`, if all the communication is within the
+     docker network (e.g., no external client outside the docker network). Otherwise, you must
+     specify the master and worker's docker host IP that client can reach out (e.g., by `-Dalluxio.worker.hostname=$(hostname -i)`).
+     This is required for the external communication between master/worker and
+     clients outside the docker network. Otherwise, clients can't connect to worker, since
+     they do not recognize the worker's container Id. It will throw error like below:
+     ```
+     Target: 5a1a840d2a98:29999, Error: alluxio.exception.status.UnavailableException: Unable to resolve host 5a1a840d2a98
+     ```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+## Verify the Cluster
+
+To verify that the services came up, check `docker ps`. You should see something like
+```shell
+$ docker ps
+CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                      NAMES
+1fef7c714d25        alluxio/{{site.ALLUXIO_DOCKER_IMAGE}}     "/entrypoint.sh work…"   39 seconds ago      Up 38 seconds                                  alluxio-worker
+27f92f702ac2        alluxio/{{site.ALLUXIO_DOCKER_IMAGE}}     "/entrypoint.sh mast…"   44 seconds ago      Up 43 seconds       0.0.0.0:19999->19999/tcp   alluxio-master
+```
+
+If you don't see the containers, run `docker logs` on their container ids to see what happened.
+The container ids were printed by the `docker run` command, and can also be found in `docker ps -a`.
+
+Visit `instance-hostname:19999` to view the Alluxio web UI. You should see one worker connected and providing
+`1024MB` of space.
+
+To run tests, enter the worker container
+
+```shell
+$ docker exec -it alluxio-worker /bin/bash
+```
+
+Run the tests
+
+```shell
+$ cd /opt/alluxio
+$ ./bin/alluxio runTests
+```
+
+To test the remote client access, for example, from the Spark cluster (python 3)
+
+```scala
+textFile_alluxio_path = "alluxio://{docker_host-ip}:19998/path_to_the_file"
+textFile_RDD = sc.textFile (textFile_alluxio_path)
+
+for line in textFile_RDD.collect():
+  print (line)
+```
+
+Congratulations, you've deployed a basic Dockerized Alluxio cluster! Read on to learn more about how to manage the cluster and make is production-ready.
+
+## Advanced Setup
+
+### Launch Alluxio with Java 11
```

Comment:
check with @ssz1997 but i believe we build our images to launch using java 11 by default. if true, then this section should be removed